assembl/static2/routes.json

```diff
@@ -30,33 +30,41 @@
   "community": "${slug}/community",
   "terms": "${slug}/terms",
   "theme": "theme/${themeId}",
-  "brightMirrorFiction": "${slug}/debate/${phase}/theme/${themeId}/fictions/${fictionId}",
+  "brightMirrorFiction":
```

Comment:
Hey @MilanBarande I kept on rollbacking this file because of the linter. And here are all the changes. Should it be changed at the end by the way ?